SchoolManagementSystemApi/Model/Teachers.cs

```diff
@@ -1,17 +1,17 @@
-﻿namespace SchoolManagementSystemApi.Model
+﻿using System.ComponentModel.DataAnnotations.Schema;
+
+namespace SchoolManagementSystemApi.Model
 {
     public class Teachers
     {
-        public Guid Id { get; set; }
-        public string FirstName { get; set; }
-        public string LastName { get; set; }
-        public string Email { get; set; }
-        public DateTime DateOfBirth { get; set; }
-        public string Gender { get; set; }
-        public string PhoneNumber { get; set; }
-        public string Address { get; set; }
-        public string? SchoolId { get; set; }
-        public string salary { get; set; }
+        public Guid UserId { get; set; }
+        [ForeignKey("UserId")]
+        public virtual ApplicationUser ApplicationUser { get; set; }
```

Comment:
@New-Anointing I thought we came to a conclusion not to use this. Even if we are, why are there two instances of UserId?

Reply:
i started this before so i decided to remove it